src/js/entities-service/entities/patient-fields.js

```diff
@@ -7,10 +7,27 @@ const TYPE = 'patient-fields';
 
 const _Model = BaseModel.extend({
   type: TYPE,
+  url() {
+    return `/api/patients/${ this.get('_patient') }/fields/${ this.get('name') }`;
+  },
+  isNew() {
+    // NOTE: This will treat the PATCH like a PUT
+    // We won't always have an ID, but never need to POST
+    return false;
+  },
   getValue() {
     const value = this.get('value');
     return isObject(value) && isEmpty(value) ? null : value;
   },
+  saveAll(attrs) {
+    attrs = extend({}, this.attributes, attrs);
```

Comment:
@paulfalgout , @nmajor25 , I think we might need to add a `uniqueId` here in `attrs`.  See my comments here: https://app.shortcut.com/roundingwell/story/34920/hra-appointment-mark-as-seen